x-pack/plugins/fleet/README.md

```diff
@@ -77,3 +77,45 @@ You need to have `docker` to run ingest manager api integration tests
 ```
 FLEET_PACKAGE_REGISTRY_DOCKER_IMAGE='docker.elastic.co/package-registry/distribution:production' FLEET_PACKAGE_REGISTRY_PORT=12345 yarn test:ftr:runner
 ```
+
+### Running Fleet Server Locally in a Container
+
+It can be useful to run Fleet Server in a container on your local machine in order to free up your actual "bare metal" machine to run Elastic Agent for testing purposes. Otherwise, you'll only be able to a single instance of Elastic Agent dedicated to Fleet Server on your local machine, and this can make testing integrations and policies difficult.
+
+_The following is adapted from the Fleet Server [README](https://github.com/elastic/fleet-server#running-elastic-agent-with-fleet-server-in-container)_
+
+1. Add the following configuration to your `kibana.dev.yml`
+
+```yml
+server.host: 0.0.0.0
+```
+
+2. Append the following option to the command you use to start Elasticsearch
+
+```
+-E http.host=0.0.0.0
+```
+
+This command should look something like this:
+
+```
+yarn es snapshot --license trial -E xpack.security.authc.api_key.enabled=true -E path.data=/tmp/es-data -E http.host=0.0.0.0
+```
+
+3. Run the Fleet Server Docker container. Make sure you include a `BASE-PATH` value if your local Kibana instance is using one. `YOUR-IP` should correspond to the IP address used by your Docker network to represent the host. For Windows and Mac machines, this should be `192.168.65.2`. If you're not sure what this IP should be, run the following to look it up:
+
+```
+docker run -it --rm alpine nslookup host.docker.internal
+```
+
+To run the Fleet Server Docker container:
+
+```
+docker run -e KIBANA_HOST=http://{YOUR-IP}/{BASE-PATH}:5601 -e KIBANA_USERNAME=elastic -e KIBANA_PASSWORD=changeme -e ELASTICSEARCH_HOST=http://{YOUR-IP}:9200 -e ELASTICSEARCH_USERNAME=elastic -e ELASTICSEARCH_PASSWORD=changeme -e KIBANA_FLEET_SETUP=1 -e FLEET_SERVER_ENABLE=1 -e FLEET_SERVER_INSECURE_HTTP=1 -p 8220:8220 docker.elastic.co/beats/elastic-agent:{VERSION}
```

Comment:
`KIBANA_HOST=http://{YOUR-IP}/{BASE-PATH}:5601`
should be
`KIBANA_HOST=http://{YOUR-IP}:5601/{BASE-PATH}`